### DISCUSSION TITLE
Button menu

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#685] [RELEASED: Components, pages and patterns]
The [button menu component](https://design-patterns.service.justice.gov.uk/components/button-menu/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by Whersi** [comment]
## Release: Button menu component 
MoJ Design System team has released a new version of the button menu component. This component no longer has a dependency on jQuery.

## Breaking changes  
There are 2 breaking changes: 

1. The button menu component no longer works as a button group. You need to change any occurrences of it to the [GOV.UK button group container](https://design-system.service.gov.uk/components/button/#grouping-buttons).

2. The component no longer switches between a button group and menu according to screen size. There are [options for displaying buttons in groups](https://design-patterns.service.justice.gov.uk/components/button-menu/).

You also need to check for [page header actions](https://design-patterns.service.justice.gov.uk/components/page-header-actions/) and the [identity bar](https://design-patterns.service.justice.gov.uk/components/identity-bar/) in your service. This is because they contain the button menu.

## New documentation 
The documentation now includes:
- when to use the component and what to consider
- how to position it on your page and group it with a GOV.UK button
- guidance on colours and content 
- examples of how the component could look in a service

View the [button menu on the MoJ Design System documentation website](https://design-patterns.service.justice.gov.uk/components/button-menu/).
